crab/cli.py

```diff
@@ -78,12 +79,20 @@ def main(command=None):
         port = env.setdefault("PORT", get_free_port())
         command = [item.replace("$PORT", port) for item in command]
 
-    if shutil.which(command[0], path=env["PATH"]) is None:
-        print('Could not find "{}" in your procfile or $PATH.'.format(command[0]))
-        exit(1)
+    if shutil.which(command[0], path=env["PATH"]) is not None:
+        return os.execvpe(command[0], command, env)
 
-    # off we go
-    os.execvpe(command[0], command, env)
+    # Is it a shell function?
+    if (
```

Comment:
I think we should follow the semantics of shell functions in that they override items in the `$PATH`, rather than only being called if they don't exist.

Reply:
Ah right, so swap the two checks?

Reply:
Indeed